content/php/concepts/math-functions/tan/tan.md

```diff
@@ -0,0 +1,30 @@
+#tan()
+
+PHP Calculate Tangent
+
+
+
+PHP’s math functions provides us with functions for calculations involving trigonometry.The tan() function in PHP is used to find the tangent of the argument parameter. The num is in radians and can handle values within the range of integer and float types.
+
+##Syntax
+
+```php
+tan(num); // Returns tan of num
+
+echo(tan(10)); // Returns 0.64836082745
+```
+##Sample Code
+
+```php
+<?php
+$num=(10);
+echo "The tangent of 10 = ".(tan($num));
+?>
+```
+The PHP math functions are part of the PHP core.  No installation is required to use these functions. 
```

Comment:
Replace this with a description of the example. Also illustrate the output within a ` ```shell ` code block.